### PR TITLE
rust/*: update path, add Tamil translation

### PR DESCRIPTION
**Pull request by kbdharun**
- [x] The page(s) are in the correct platform directories: `common`.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by EmilySeville7cfg — APPROVED** [review]
No fixes found, approved.
Thanks for a contribution! ❤️

**Comment by kbdharun** [comment]
Can this be merged? Somebody?

File: pages.zh/common/rustfmt.md [hunk 1]
```diff
@@ -5,12 +5,12 @@
 
 - 格式化文件，就地覆盖原始文件：
 
-`rustfmt {{source.rs}}`
+`rustfmt {{路径/到/源.rs}}`
```
Comment (blueskyson) [inline pages.zh/common/rustfmt.md:8]: "source file" sounds more natural than `source` in Chinese.
```suggestion
`rustfmt {{路径/到/源文件.rs}}`
```

File: pages.zh/common/rustfmt.md [hunk 2]
```diff
@@ -5,12 +5,12 @@
...
 
 - 检查文件的格式并在控制台上显示所有更改：
 
-`rustfmt --check {{source.rs}}`
+`rustfmt --check {{路径/到/源.rs}}`
```
Comment (blueskyson) [inline pages.zh/common/rustfmt.md:12]:
```suggestion
`rustfmt --check {{路径/到/源文件.rs}}`
```

File: pages.zh/common/rustfmt.md [hunk 3]
```diff
@@ -5,12 +5,12 @@
...
 
 - 格式化之前，备份所有修改过的文件（原始文件的扩展名为 `.bk`）：
 
-`rustfmt --backup {{source.rs}}`
+`rustfmt --backup {{路径/到/源.rs}}`
```
Comment (blueskyson) [inline pages.zh/common/rustfmt.md:16]:
```suggestion
`rustfmt --backup {{路径/到/源文件.rs}}`
```

**Review by blueskyson — APPROVED** [review]

**Pull request merged**